packages/stateless/src/lib/provider/provider.interface.ts

```diff
@@ -56,11 +56,40 @@ export interface ISendMessageSuccessResponse {
   date?: string;
 }
 
+export enum EmailEventStatusEnum {
+  OPEN = 'open',
+  DELIVERY = 'delivery',
+  BOUNCE = 'bounce',
+  DROPPED = 'dropped',
+  CLICK = 'click',
+}
```

Comment:
```suggestion
export enum EmailEventStatusEnum {
  OPEN = 'open',
  DELIVERY = 'delivery',
  BOUNCE = 'bounce',
  DROP = 'drop',
  CLICK = 'click',
}
```
I would suggest to follow the same convention for the naming. If the status is going to be represent by infinitives, all in infinitives. Another option is to move them all to past participles, that can maybe give a closer explanation of the status:
```suggestion
export enum EmailEventStatusEnum {
  OPENED = 'opened',
  DELIVERED = 'delivered',
  BOUNCED = 'bounced',
  DROPPED = 'dropped',
  CLICKED = 'clicked',
}
```